junebug/logging_service.py

```diff
@@ -72,13 +70,13 @@ def _log_to_file(self, event):
             data['instance'] = repr(failure.value)
             data['stack'] = failure.stack
 
-        data['message'] = log.textFromEventDict(event)
-
         self.logfile.write(json.dumps(data))
 
     def __call__(self, event):
         if self.log_context_sentinel in event:
             return
+        if self.worker_id not in event.get('system', ''):
+            return
```

Comment:
I'm not sure that we would always expect the `<worker_id>` to be first in the system. What if we decide to organise it as `transports,<worker_id>` in future, to group all the transports, if we wanted to also return say application logs as well as transport logs in the future, because then we would also want `applications,<worker_id>`. 

I'm not sure I understand about the `.` after the worker id? For our current case, the `system` would just be `<worker_id>`, no `.`.